npm/pkg/dataplane/dataplane_windows_test.go

```diff
@@ -0,0 +1,21 @@
+package dataplane
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/azure-container-networking/common"
+	"github.com/Azure/azure-container-networking/npm/pkg/dataplane/ipsets"
+	dptestutils "github.com/Azure/azure-container-networking/npm/pkg/dataplane/testutils"
+)
+
+func TestRefreshAllPodEndpoints(t *testing.T) {
```

Comment:
can we delete this file. I started writing this but its not important and will conflict with #1652

Reply:
deleted